test/Microsoft.AspNetCore.Authentication.Test/Cookies/CookieMiddlewareTests.cs

```diff
@@ -851,68 +740,85 @@ private Task SignOutAsWrong(HttpContext context)
         }
 
         [Fact]
-        public async Task MapWillNotAffectChallenge()
+        public async Task MapWillAffectChallenge()
```

Comment:
Be careful, it's most likely not a wanted change: if `/page` is configured as the callback path and the user is actually redirected to `/login/page`, all he'll get is a 404 response, as no login page will handle the request.

I assume it similarly impacts all the RAT-based middleware that use callback paths and have built-in endpoints? (OIDC, OAuth2, Twitter...)